boot/aether/project.clj

```diff
@@ -16,4 +16,5 @@
   :dependencies [[org.clojure/clojure      "1.6.0"  :scope "compile"]
                  [boot/base                ~version :scope "provided"]
                  [boot/pod                 ~version :scope "compile"]
-                 [com.cemerick/pomegranate "0.3.0"  :scope "compile"]])
+                 [com.cemerick/pomegranate "0.3.0"  :scope "compile"]
+                 [org.apache.maven.wagon/wagon-http "2.9"]])
```

Comment:
This entry is just to bump up the version already included by pomegranite - so sounds like yes, should have the same scope